Makefile

```diff
@@ -414,13 +417,17 @@ venv:
 	$(PYTHON) -m pip install -U pip
 	$(PYTHON) -m pip install -U -r requirements.txt
 
+setup-audio:
+	$(AUDIO_EXTRACT) -o $(EXTRACTED_DIR) -v $(VERSION) --read-xml
+
 setup: venv
 	$(MAKE) -C tools
 	$(PYTHON) tools/decompress_baserom.py $(VERSION)
 	$(PYTHON) tools/extract_baserom.py $(BASEROM_DIR)/baserom-decompressed.z64 --oot-version $(VERSION) -o $(EXTRACTED_DIR)/baserom
 	$(PYTHON) tools/extract_incbins.py $(EXTRACTED_DIR)/baserom --oot-version $(VERSION) -o $(EXTRACTED_DIR)/incbin
 	$(PYTHON) tools/msgdis.py $(VERSION)
 	$(PYTHON) extract_assets.py -v $(VERSION) -j$(N_THREADS)
+	$(MAKE) setup-audio
```

Comment:
recursive make considered harmful

seriously though, it has a real cost (especially with our complex makefile with tons of build targets and shell commands) and kinda feels ugly. It would be better to do something like:
```
setup-audio: setup-baserom
        $(AUDIO_EXTRACT) -o $(EXTRACTED_DIR) -v $(VERSION) --read-xml

setup: setup-assets setup-text setup-audio
```
although that would be a larger change

Reply:
I need some convenient way to test audio extraction without running everything else, so I guess I'll go with the larger change

Reply:
Actually this suggestion is slower for my needs since depending on setup-tools causes setup-audio to recurse into the tools makefile, triggering some ZAPD stuff that I'd rather not deal with when trying to quickly test something.

Reply:
Actually on second thought I think the larger change might run everything else too. Is it so bad though to copy-paste the command from the make output if you want to retest after making a change?

Reply:
It's pretty bad yeah, what with the venv and version dependence, I'd rather not retype the command.

Reply:
(discussed offline and compromised)